client/src/components/topics.tsx

```diff
@@ -50,7 +58,7 @@ function Topics(args: {
   // if not mobile -> hide history button
 
   const topicButtons = topicQuestions.map((tq, i) => {
-    return shouldDisplayPortrait() ? (
+    return shouldDisplayPortrait() && mentorType !== "CHAT" ? (
```

Comment:
same issue as mentioned in last review:

We want to maximize reusability of this (pure presentation) topics component. The component will be more reusable if it takes a property like `includeHistoryTopic: boolean` instead of it owning the business logic of deciding whether to show History based on mentorType

Reply:
Sure, same reason but I already fixed passing styling props